hack/local-up-cluster.sh

```diff
@@ -376,7 +376,6 @@ cleanup()
   if [[ "${PRESERVE_ETCD}" == "false" ]]; then
     [[ -n "${ETCD_DIR-}" ]] && kube::etcd::clean_etcd_dir
   fi
-  exit 0
```

Comment:
do we know why this was added previously?

Reply:
cut-n-paste from somewhere?